src/main/java/com/kryptnostic/conductor/rpc/ResultSetAdapterFactory.java

```diff
@@ -75,12 +90,21 @@ public static Entity mapRowToEntity( Row row, Set<PropertyType> properties ) {
 	public static SetMultimap<FullQualifiedName, Object> mapRowToObject( Row row, Collection<PropertyType> properties ) {
 		SetMultimap<FullQualifiedName, Object> map = HashMultimap.create();
 		properties.forEach(property -> {
-			Object value = row.getObject( "value_" + property.getTypename() );
+			Object value = getObject( row, CassandraEdmMapping.getCassandraType( property.getDatatype() ), "value_" + property.getTypename() );
```

Comment:
This probably legacy badness from way back-- do you remember why we're prefixing `value_`?